client/shared/helpers/user.coffee

```diff
@@ -31,40 +31,32 @@ module.exports =
 
   subscribeToLiveUpdate: (options = {}) ->
     return unless AbilityService.isLoggedIn()
-    Records.messageChannel.remote.post('subscribe', options).then (subscriptions) ->
-      _.each subscriptions, (subscription) ->
-        PrivatePub.sign(subscription)
-        PrivatePub.subscribe subscription.channel, (data) ->
-          liveUpdateAction(data)       if data.action?
-          liveUpdateVersion(data)      if data.version?
-          liveUpdateMemo(data)         if data.memo?
-          liveUpdateEvent(data)        if data.event?
-          liveUpdateNotification(data) if data.notification?
-          Records.import(data)
-
-liveUpdateAction = (data) ->
-  if data.action == 'logged_out' && !AppConfig.loggingOut
-    ModalService.open 'SignedOutModal', -> preventClose: true
-
-liveUpdateVersion = (data) ->
-  FlashService.update 'global.messages.app_update', {version: data.version}, 'global.messages.reload', hardReload
 
-liveUpdateMemo = (data) ->
-  switch data.memo.kind
-    when 'comment_destroyed'
-      if comment = Records.comments.find(memo.data.comment_id)
-        comment.destroy()
-    when 'comment_updated'
-      Records.comments.import(memo.data.comment)
-      Records.import(memo.data)
-    when 'comment_unliked'
-      if comment = Records.comments.find(memo.data.comment_id)
-        comment.removeLikerId(memo.data.user_id)
-
-liveUpdateEvent = (data) ->
-  data.events = [] unless _.isArray(data.events)
-  data.events.push(data.event)
-
-liveUpdateNotification = (data) ->
-  data.notifications = [] unless _.isArray(data.notifications)
-  data.notifications.push(data.notification)
+    AppConfig.cable.subscriptions.create { channel: "ApplicationChannel" },
+      received: (data) ->
+        if data.version?
+          FlashService.update 'global.messages.app_update', {version: data.version}, 'global.messages.reload', hardReload
+
+    AppConfig.cable.subscriptions.create { channel: "UserChannel" },
+      received: (data) ->
+        if data.action? && !AppConfig.loggingOut
+          AppConfig.loggingOut = true
+          ModalService.open 'SignedOutModal', -> preventClose: true
+        Records.import(data)
+
+    _.each Session.user().groups(), (group) ->
+      AppConfig.cable.subscriptions.create { channel: "GroupChannel", group_id: group.id },
+        received: (data) ->
+          console.log("received ", data)
+          if data.memo?
+            switch data.memo.kind
+              when 'comment_destroyed'
+                if comment = Records.comments.find(memo.data.comment_id)
+                  comment.destroy()
+              when 'comment_updated'
```

Comment:
I think we just want to create a comment_updated event and publish it the same way as the other events.